doc-content/enterprise-only/openshift/offline-params.adoc

```diff
@@ -0,0 +1,15 @@
+. If your OpenShift environment does not have outgoing access to the public Internet, set the following parameters:
++
+** *Maven mirror URL*(`MAVEN_MIRROR_URL`): The URL for the Maven mirror repository that you set up in <<offline-repo-proc>>. This URL must be accessible from a pod in your OpenShift environment.
+** *Maven mirror of*(`MAVEN_MIRROR_OF`): The value that determines which artifacts are to be retrieved from the mirror.
+ifndef::params_offline_central[]
+The default value is `external:*`; with this value, Maven retrieved every required artifact from the mirror and does not query any other repositories.
```

Comment:
"retrieved" should be "retrieves"

Reply:
It might also be worthwhile to link to the "Advanced Mirror Specification" section here: https://maven.apache.org/guides/mini/guide-mirror-settings.html